how-to-guides/restart-run-from-checkpoint/notebooks/neptune_save_restart_run_from_checkpoint.ipynb

```diff
@@ -0,0 +1,523 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "rEp1Qqj-OkTW"
+   },
+   "source": [
+    "# How to restart a run from checkpoint"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Introduction\n",
+    "Resuming ML experiments from checkpoints is essential to ensure progress is not lost in scenarios such as server disruptions or failures. \n",
+    "\n",
+    "By the end of this guide, you learn how to resume your experiment from a saved checkpoint.\n",
```

Comment:
@TytusKalicki - you can suggest changes as below
![image](https://github.com/neptune-ai/examples/assets/41324509/cd510a46-c980-4086-a793-5e15fac82865)

It will render as:
```suggestion
    "By the end of this guide, you'll learn how to resume your experiment from a saved checkpoint.\n",
```

This makes it easier to apply those changes straight from GitHub without having to make changes locally and pushing them again :)